src/game/ctrl.rs

```diff
@@ -129,9 +129,9 @@ impl GameController {
     }
 
     pub fn poll_analog2d(&self, motion: &Analog2d) -> Vec2f {
-        match motion {
-            &Analog2d::Mouse(sensitivity) => self.mouse_rel * sensitivity,
-            &Analog2d::Gestures(
+        match *motion {
```

Comment:
Is the any reason this is preferred? I have no preference on way or another really. Is it just because you end up typing less (one `*` instead of a bunch of `&`)?